components/CopyButton.js

```diff
@@ -7,11 +7,13 @@ export default class CopyButton extends Component {
   static propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
+    iconDescription: PropTypes.string,
     feedback: PropTypes.string,
     feedbackTimeout: PropTypes.number,
     onClick: PropTypes.func,
   };
   static defaultProps = {
+    iconDescription: 'Provide an icon description',
```

Comment:
Should this actually be the description of the copy icon instead? User's don't get to chose which icon goes here